test/fixtures/rules/std_rule_cases/L019.yml

```diff
@@ -198,3 +198,79 @@ leading_comma_fixing_flows_around_comments:
     rules:
       L019:
         comma_style: leading
+
+trailing_comma_move_past_several_comment_lines:
+  fail_str: |
+    SELECT
+    COUNT(1) AS campaign_count,
+    state_user_v_peer_open
+    ,business_type
+
+    -- The following is the slope of the regression line. Note that CORR (which is the Pearson's correlation
+    --  coefficient is symmetric in its arguments, but since STDDEV_POP(open_rate_su) appears in the
+    --  numerator this is the slope of the regression line considering STDDEV_POP(open_rate_su) to be
+    --  the "y variable" and uses_small_subject_line to be the "x variable" in terms of the regression line.
+    ,SAFE_DIVIDE(SAFE_MULTIPLY(CORR(open_rate_su, uses_small_subject_line), STDDEV_POP(open_rate_su)), STDDEV_POP(uses_small_subject_line))
+
+    FROM
+    global_actions_states
+  fix_str: |
+    SELECT
+    COUNT(1) AS campaign_count,
+    state_user_v_peer_open,
+    business_type,
+
+    -- The following is the slope of the regression line. Note that CORR (which is the Pearson's correlation
+    --  coefficient is symmetric in its arguments, but since STDDEV_POP(open_rate_su) appears in the
+    --  numerator this is the slope of the regression line considering STDDEV_POP(open_rate_su) to be
+    --  the "y variable" and uses_small_subject_line to be the "x variable" in terms of the regression line.
+    SAFE_DIVIDE(SAFE_MULTIPLY(CORR(open_rate_su, uses_small_subject_line), STDDEV_POP(open_rate_su)), STDDEV_POP(uses_small_subject_line))
+
+    FROM
+    global_actions_states
+  configs:
+    core:
+      # Set runaway_limit=1 to verify the fix only requires one pass. In an
+      # earlier version, the comma before "SAFE_DIVIDE()" was being moved one
+      # line per pass. Too lazy!
+      runaway_limit: 1
+
+
+leading_comma_move_past_several_comment_lines:
```

Comment:
I added this new test for the "leading comma" case to complement the new "trailing comma" case above. For this test, no code changes were required -- I guess L019 was already handling this case efficiently. 👍🏽